src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts

```diff
@@ -278,11 +278,12 @@ export class MarkdownToolbarComponent extends AngularDisposable {
 		const triggerHeight = triggerElement.offsetHeight;
 		const triggerWidth = triggerElement.offsetWidth;
 		const dialogProperties = { xPos: triggerPosX, yPos: triggerPosY, width: triggerWidth, height: triggerHeight };
+		const dialogPosition = window.innerHeight - triggerPosY < 250 ? 'above' : 'below';
```

Comment:
250 is a magic number. What does it refer to? How can we make this generic?

Reply:
the height of the dialog + the arrow at the bottom of the dialog - will also look for this property